orangecontrib/snom/preprocess/phase_unwrap.py

```diff
@@ -6,17 +6,19 @@
 
 from orangecontrib.spectroscopy.preprocess.utils import (
     SelectColumn,
-    CommonDomainOrderUnknowns,
+    CommonDomain,
 )
 
 
-class _PhaseUnwrapCommon(CommonDomainOrderUnknowns):
```

Comment:
Why did you make this change?

The old one ensures that you `transformed` function will not get any unknown values (handles them externally) and also handles that wavenumbers are put in the correct order (if they got mixed in Select Columns) or something.

If neither is needed, remove the comment too.